asset-transfer-events/application-javascript/app.js

```diff
@@ -0,0 +1,519 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict';
+
+/**
+ * Application that shows events when creating and updating an asset
+ *   -- How to register a contract listener for chaincode events
+ *   -- How to get the chaincode event name and value from the chaincode event
+ *   -- How to retrieve the transaction and block information from the chaincode event
+ *   -- How to register a block listener for full block events
+ *   -- How to retrieve the transaction and block information from the full block event
+ *   -- How to register to recieve private data associated with transactions when
+ *      registering a block listener
+ *   -- How to retreive the private data from the full block event
+ *   -- The listener will be notified of an event at anytime. Notice that events will
+ *      be posted by the listener after the application activity causing the ledger change
+ *      and during other application activity unrelated to the event
+ *
+ * To see the SDK workings, try setting the logging to be displayed on the console
+ * before executing this application.
+ *        export HFC_LOGGING='{"debug":"console"}'
+ * See the following on how the SDK is working with the Peer's Event Services
+ * https://hyperledger-fabric.readthedocs.io/en/latest/peer_event_services.html
+ *
+ * See the following for more details on using the Node SDK
+ * https://hyperledger.github.io/fabric-sdk-node/release-2.2/module-fabric-network.html
+ */
+
+// pre-requisites:
+// - fabric-sample two organization test-network setup with two peers, ordering service,
+//   and 2 certificate authorities
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh up createChannel -ca
+//
+// - Use the asset-transfer-events/chaincode-javascript chaincode deployed on
+//   the channel "mychannel". The following deploy command will package, install,
+//   approve, and commit the javascript chaincode, all the actions it takes
+//   to deploy a chaincode to a channel.
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh deployCC -ccn events -ccl javascript -ccep "OR('Org1MSP.peer','Org2MSP.peer')"
+//
+// - Be sure that node.js is installed
+//         ===> from directory /fabric-samples/asset-transfer-sbe/application-javascript
+//         node -v
+// - npm installed code dependencies
+//         ===> from directory /fabric-samples/asset-transfer-sbe/application-javascript
+//         npm install
+// - to run this test application
+//         ===> from directory /fabric-samples/asset-transfer-sbe/application-javascript
+//         node app.js
```

Comment:
This should refer to fabric-samples/asset-transfer-events/application-javascript instead and I wouldn't assume that it's in the root directory so I suggest dropping the leading /